packages/api/scripts/add-mock-deals.js

```diff
@@ -2,9 +2,10 @@
  * Add some mock deals to your database for your existing content.
```

Comment:
Fauna exposed the GraphQL interface, this file can be removed entirely - it does not work.